godotproject/Objects/Rocket/doorWallBot.gd

```diff
@@ -0,0 +1,18 @@
+extends CollisionShape2D
+
+
+var tongue_stick_velocity = Vector2(0, -15) # FIXME ideally should be get_parent().speed
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+#func _process(delta):
+#	pass
```

Comment:
```suggestion
```

Reply:
hmmmm I thought I deleted this whole file. I'll see why its still showing as there